web/celery_app/tasks.py

```diff
@@ -10,13 +10,15 @@ def send_notification_email(name, email, text_box):
 
     message = (
         'An error report has been submitted by {}'
-        ' who provided the following email {}. The '
-        'following information was sent {}'
+        ' who provided the following email {}. '
+        '\nThe following information was sent:\n\n {}'
+        '\n'
+        '\n https://errorreports.mantidproject.org/admin/'
     ).format(name, email, text_box)
 
     send_mail(subject,
               message,
-              'mantidproject@gmail.com',
+              'error-reports@mantidproject.org',
```

Comment:
Maybe we shouldn't hard code this either? Can we set it as a configurable variable too?